src/libraries/System.Private.CoreLib/src/System/IO/StringReader.cs

```diff
@@ -175,38 +180,40 @@ public override string ReadToEnd()
         //
         public override string? ReadLine()
         {
-            if (_s == null)
+            string? s = _s;
+            if (s == null)
             {
-                throw new ObjectDisposedException(null, SR.ObjectDisposed_ReaderClosed);
+                ThrowObjectDisposedException_ReaderClosed();
             }
+            var pos = _pos;
```

Comment:
nit: it's not clear what type is it
```suggestion
            int pos = _pos;
```